create_md.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Something is wrong with this python script. See logs. The check failed.

Reply:
![image](https://github.com/user-attachments/assets/98ff53bb-7581-4901-9cd3-45a8df8a2248)